.github/workflows/build.yaml

```diff
@@ -263,30 +180,73 @@ jobs:
         with:
           check_name: Indy Node ${{ matrix.module }} Test Report for slice ${{ matrix.slice }}/${{ strategy.job-total }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          report_paths: "*-test-results.xml"
+          report_paths: "/indy-node/*-test-results.xml" 
 
       - name: Upload Detailed Test Failure Results
         # The test runner only emits the detailed test results if the tests fail.
         if: (steps.node-test.outcome == 'failure') && failure()
         uses: actions/upload-artifact@v2
         with:
           name: detailed-test-result-slice-${{ matrix.slice }}
-          path: test-result-node-${{ matrix.slice }}.txt
+          path: /indy-node/test-result-node-${{ matrix.slice }}.txt
           retention-days: 5
 
+  remove_test_image:
+    name: Remove Test image from GHCR
+    needs: [workflow-setup, indy_node_tests]
+    if: always()
+    runs-on : ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run a one-line script
+        run: |
+            curl -u pschlarb:${{ secrets.API }} -H "Accept: application/vnd.github.v3+json" \
+            $GITHUB_API_URL/user/packages/container/node-build/versions > apiResponse.json
+      - name: Cat JSON
+        run: |
+            cat apiResponse.json
+      - name: Run Pyhton Script
+        id: pversion
+        env:
+          SHORT_SHA: ${{ needs.workflow-setup.outputs.shortSHA }}
+        run: |
+          echo "::set-output name=packageVersion::$(python .github/workflows/getGHCRPackageID.py $SHORT_SHA)"
+      - name: Remove Docker Container or Package with sha
+        env: 
+          PACKAGE_VERSION: ${{ steps.pversion.outputs.packageVersion }}
+        run: |
+            curl -u pschlarb:${{ secrets.API }} -H "Accept: application/vnd.github.v3+json" -X DELETE $GITHUB_API_URL/user/packages/container/node-build/versions/$PACKAGE_VERSION | grep "last tagged"
+            if [ $? -eq 0 ];
+            then 
+              echo $(curl -u pschlarb:${{ secrets.API }} -X DELETE -H"Accept: applicatiov/vnd.github.vs3+json" $GITHUB_API_URL/user/packages/container/node-build);
+            else 
+              echo done;
+            fi;
+
```

Comment:
Is there no way to get this process to dynamically work on different repositories rather than hardcoding the username, or is that what the `GITHUB_REPOSITORY_OWNER` is going to be used for?

Reply:
> It was neccessary because with `github.repository` the container would have another level ( [USER|ORG]/[repo]/node-build ), which couldn't be accessed by the package api.

And yes it is going to be used for dynamically use of it.